marklogic-data-hub/src/test/java/com/marklogic/hub/EndToEndFlowTests.java

```diff
@@ -783,14 +830,25 @@ private void testInputFlowViaMlcp(String prefix, String fileSuffix, DatabaseClie
         }
         logger.error(mlcpRunner.getProcessOutput());
 
-        assertEquals(finalCounts.stagingCount, getStagingDocCount());
-        assertEquals(finalCounts.finalCount, getFinalDocCount());
-        assertEquals(finalCounts.tracingCount, getTracingDocCount());
-        assertEquals(finalCounts.jobCount, getJobDocCount());
+        int stagingCount = getStagingDocCount();
```

Comment:
curious about this change -- it's fine, just curious

Reply:
it allowed me to put a conditional breakpoint when finalCount != finalCounts.finalCount. It was merely there for my debugging purposes.

Reply:
cool